src/components/search/styles.js

```diff
@@ -0,0 +1,30 @@
+import {makeStyles} from '@material-ui/core/styles';
+
+export default makeStyles((theme)=>({
+    listItem: {
+        lineHeight:2,
+        width:'100%',
+        '&:hover': {
+            backgroundColor:'rgba(0,0,0,.025)',
+            cursor:'pointer',
+        },
+    featureItem: {
+        lineHeight:2,
+
+        '&:hover':{
+            textDecoration: 'underline',
+        }
+        
+    },
+    myBar: {
+      margin:'0 auto',
+      textAlign: 'center',
+      lineHeight: '2.5',
+  },
+    },
+      [theme.breakpoints.down('sm')]:{
+        mainContainer:{
+          display:'block'
+        }
+      }
+}))
```

Comment:
It seems this is the only use of the `material-ui` dependencies... It's a lot to add just for styling. Is it possible to use the LESS styling mechanism already in place provided by Craco? If not, that's OK, but it would be preferrable to avoid the additional weight of the `material-ui` dependencies if possible.

Reply:
I'm going to second this. Unless I'm missing something, this seems like it could be handled in LESS without bringing in new deps. a full component library for a little theming seems pretty heavy.